source/stylesheets/_tab-stack.scss.erb

```diff
@@ -259,66 +256,29 @@
 							}
 
 							.credit-per-min {
-								text-align: right;
-								white-space: nowrap;
-
-								.value, .unit {
-									font-size: 16px;
-									font-weight: bold;
-									font-style: normal;
-									font-stretch: normal;
-									line-height: initial;
-									letter-spacing: 0.07px;
-									color: $def-darkgrey;
-									@include ellipsis-oneliner;
-								}
+								font-size: 16px;
+								font-weight: bold;
+								color: #201b22;
 							}
 						}
 
 						.separator {
 							height: 1px;
-							margin-bottom: 3px;
+							@include margin-y(8px);
 							background-color: $def-verylightgrey;
 						}
 
 						.bottom {
 							display: flex;
-							align-items: center;
-							width: 100%;
-
-							.cpu {
```

Comment:
I saw a lot of deletions from CSS, the new layout doesn't need them at all?

Reply:
Nope, they were simple inline texts not needing any special styling.